src/Microsoft.OData.Edm/Csdl/EdmxReader.cs

```diff
@@ -357,9 +358,9 @@ private List<CsdlModel> LoadAndParseReferencedEdmxFiles(Version mainEdmxVersion)
                     continue;
                 }
 
-                if (tmp.Uri != null
-                    && (tmp.Uri.EndsWith(CoreVocabularyConstants.VocabularyUrlSuffix, StringComparison.Ordinal))
-                        || tmp.Uri.EndsWith(CapabilitiesVocabularyConstants.VocabularyUrlSuffix, StringComparison.Ordinal))
+                if (tmp.Uri != null && (tmp.Uri.EndsWith(CoreVocabularyConstants.VocabularyUrlSuffix, StringComparison.Ordinal) ||
+                    tmp.Uri.EndsWith(CapabilitiesVocabularyConstants.VocabularyUrlSuffix, StringComparison.Ordinal) ||
+                    tmp.Uri.EndsWith(CommunityVocabularyConstants.VocabularyUrlSuffix, StringComparison.Ordinal)))

```

Comment:
Will we need to add another `||`, if there is another community vocabulary, such as Display?  We shall load, and optionally cache a community vocabulary.